core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/HolidayCalendarEditViewModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.ustadmobile.core.viewmodel
+
+import com.ustadmobile.lib.db.entities.HolidayCalendar
+
+data class HolidayCalendarEditUiState(
+    val holidayCalendar: HolidayCalendar? = null,
+    val fieldsEnabled: Boolean = true,
+    val calendarList: List<HolidayCalendar>? = emptyList()
```

Comment:
Like HolidayCalendarDetail, this should be holidayList and it should be a list of Holiday, not HolidayCalendar.